lib/api/1.1/southbound/profiles.js

```diff
@@ -52,17 +52,13 @@ function profilesRouterFactory (
 
                 return profileApiService.getNode(macAddresses, options)
                 .then(function (node) {
-                    if (node.newRecord) {
-                        presenter(req, res).renderProfile('redirect.ipxe');
-                    } else {
-                        return profileApiService.renderProfileFromTaskOrNode(node, 'compute')
-                        .then(function (render) {
-                            presenter(req, res).renderProfile(render.profile, render.options, 200, render.context);
-                        });
-                    }
+                    return profileApiService.getProfileFromTaskOrNode(node, 'compute')
+                    .then(function (render) {
+                        presenter(req, res).renderProfile(render.profile, render.options, 200, render.context);
+                    });
                 });
             } else {
-                presenter(req, res).renderProfile('redirect.ipxe');
+                presenter(req, res).renderProfile('redirect.ipxe', { isLookUpNotChecked: true });
```

Comment:
A little suggestion, how about renaming the variable to `ignoreLookup` or `ignoreLookupFailure`?